### DISCUSSION TITLE
Is there a way to disable stack trace in development?

### DISCUSSION THREAD

**Discussion by eddeee888** [vercel/next.js#12407] [Help]
Does anyone know how to see the error page instead of the stacktrace in dev? 

According to the documentation, it will show the proper error page in production: https://nextjs.org/docs/advanced-features/custom-error-page#customizing-the-error-page

Is there a flag to see the error pages in dev?

**Comment by leerob** [comment]
If you run `next build && next start` this will emulate production instead of `next dev`.

**Reply by eddeee888** [reply]
Thanks for the tip! Does it mean I have to restart the server every time I make a change ? i.e. losing hot module reload

**Reply by leerob** [reply]
Yes. I haven't tried this, but maybe setting `NODE_ENV=production` will work. I'm not sure the full criteria for showing that page.

**Reply by eddeee888** [reply]
If I'm looking at the code correctly, it looks like it's hardcoded when starting dev server so I guess there's no flag atm to do this :( 
https://github.com/zeit/next.js/blob/960c18da53af89115c7e96dc8188646a51c53c15/packages/next/server/next-dev-server.ts#L50

**Reply by rheaditi** [reply]
Would be great to have a flag - this would be most useful when we're trying to _build_ custom error pages & corresponding logic!

`next build && next start` on large projects could be fairly slow for development 😅
